api/external/infra_proxy/response/nodes.proto

```diff
@@ -43,3 +43,28 @@ message UpdateNode {
     // Node name.
     string name = 1;
 }
+
+message Node {
+    // Node ID.
+    string node_id = 1;
+    // Node name.
+    string name = 2;
+    // Node environment.
+    string environment = 3;
+    // Node policy name.
+    string policy_name = 4;
+    // Node policy group.
+    string policy_group = 5;
+    // Node run-list.
+    repeated string run_list = 6;
+    // Node tags.
+    repeated string tags = 7;
+    // Node automatic attributes JSON.
```

Comment:
@vsingh-msys Are these "Node automatic attributes" in JSON format? Also are these automatic attributes specifically about the node? Based on your reply, I have recommendations for rephrasing the commented lines (62-68) for better clarity.

Reply:
> Are these "Node automatic attributes" in a JSON format?
Yes, Node automatic attributes are in JSON format.

> Also, are these automatic attributes specifically about the node?

Yes, here is meant only for Node only but Environments, Roles also have similar kind of attributes.

Reply:
@vsingh-msys Thank you for clarifying! I do not have any recommended changes based on your response.